src/snorkel_extractor_memex/Snorkel_Pipeline_Version_NB3.ipynb

```diff
@@ -46,13 +46,13 @@
     "import numpy as np\n",
     "\n",
     "#For PostgreSQL\n",
-    "postgres_location = 'postgresql://jdunnmon:123@localhost:5432'\n",
+    "#postgres_location = 'postgresql://jdunnmon:123@localhost:5432'\n",
     "\n",
     "#postgres_db_name = 'es_locs_small'\n",
-    "os.environ['SNORKELDB'] = os.path.join(postgres_location,postgres_db_name)\n",
+    "#os.environ['SNORKELDB'] = os.path.join(postgres_location,postgres_db_name)\n",
     "\n",
     "#For local PostgreSQL\n",
-    "#os.environ['SNORKELDB'] = 'postgres:///es_locs_small'\n",
+    "os.environ['SNORKELDB'] = 'postgres:///es_locs_small'\n",
```

Comment:
Can we move changes like this to a local branch?  Or maybe just keep folks' configs in comments for now?  B/c this is on master, don't want to change too much...

Reply:
Sure.